crypto/evp/evp_fetch.c

```diff
@@ -385,21 +386,26 @@ static int evp_set_default_properties(OPENSSL_CTX *libctx,
     return 0;
 }
 
-int EVP_set_default_properties(OPENSSL_CTX *libctx, const char *propq)
+int evp_set_default_properties_intern(OPENSSL_CTX *libctx, const char *propq,
+                                      int loadconfig)
```

Comment:
It is really confusing to have evp_set_default_properties EVP_set_default_properties and evp_set_default_properties_intern. Perhaps rename evp_set_default_properties to evp_set_parsed_default_properties and (optionally) evp_set_default_properties_intern to evp_set_default_properties.

Reply:
I renamed `evp_set_default_properties` to `evp_set_parsed_default_properties` but left `evp_set_default_properties_intern` as it is.

Reply:
One reason (unwritten here, my bad) why I suggested the rename of the _intern function was that the suffix is not used anywhere else - we usually use _int. But anyway it is fine to keep it as is.

Reply:
I renamed it to use _int